packages/grid/helm/helm.py

```diff
@@ -140,6 +161,14 @@ def main() -> None:
 
     # Load the multi-doc yaml file
     try:
+        # append custom docs
+        input_data = "\n---\n".join(
+            [
+                input_data,
+                ingress_with_tls(),
```

Comment:
NITPICK: We could add  it in devspace.yaml , in ingress deployment manifests?

Reply:
Specifically not added it in devspace as it's a dev tool and primary use will be for local dev & we'll be moving away from generating helm charts from it. Aditionally, devspace doesn't offer any such feature as well, unless we use profiles which generates a whole different chart.

Reply:
makes sense